src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_command_type.py

```diff
@@ -99,7 +100,7 @@ def keyvault_command_handler(command_args):
                     return _encode_hex(transform_result(result))
 
                 # otherwise handle based on return type of results
-                if isinstance(result, AzureOperationPoller):
+                if isinstance(result, (AzureOperationPoller, LROPoller)):
```

Comment:
I feel like I see this double list too many times. I would says that cli-core should have a:
```python
POLLER_CLASSES=(AzureOperationPoller, LROPoller)
```
and you import and use that everywhere. advantage is you avoid duplicating this list, and you avoid the modules to import both classes just a simple test.